internal/stack/stacks.go

```diff
@@ -158,6 +162,13 @@ func (p *stackParser) parseStack(line string) (Stack, error) {
 			// Just skip it.
 			continue
 		}
+		if elidedFramesMatcher.MatchString(line) {
```

Comment:
rather than looking for regex on each line (which could get slow for larger stack traces), I'd recommend looking for the specific prefix + suffix which is much cheaper:
```
if strings.HasPrefix(line, "...") && strings.HasSuffix(line, " frames elided...") {
  ...
}
```

If those match, we can do the regex match, though I also wonder if those checks are enough to treat this as elided frames

Reply:
agreed, if we do these checks, I don't believe the regex is necessary.